Cargo.toml

```diff
@@ -1,20 +1,21 @@
 [workspace]
 resolver = "2"
 members = [
-	"layershellev",
-	"iced_layershell",
-	"iced_layershell_macros",
-	"iced_sessionlock",
-	"starcolorkeyboard",
-	"sessionlockev",
-	"waycrate_xkbkeycode",
-	"iced_examples/*",
+  "layershellev",
+  "iced_layershell",
+  "iced_layershell_macros",
+  "iced_sessionlock",
+  "iced_sessionlock_macros",
+  "starcolorkeyboard",
+  "sessionlockev",
+  "waycrate_xkbkeycode",
+  "iced_examples/*",
 ]
 
 [workspace.package]
 authors = [
-	"Decodertalkers <aakari@tutanota.com>",
-	"Aakash Sen Sharma <aakashsensharma@gmail.com>",
+  "Decodertalkers <aakari@tutanota.com>",
```

Comment:
you can add your name here

Reply:
It's fine, no need, I'm just happy to help out whenever I can, and I don't know If I will be much of use on the actual logic parts of the project since I'm still relatively new to the wayland-specific application building :) So I feel a bit uneasy about adding myself as an author/maintainer. 

Metaprogramming is just fun for me so it wasn't a chore for me to do at all.